examples/zone-printer/README.md

```diff
@@ -1,258 +1,244 @@
-spec with two containers. One container mounts the aforementioned
-ConfigMap, executes the script in the ConfigMap and writes the
-output to `/usr/share/nginx/html/index.html`. The other container
-runs a standard nginx server that reads from `/usr/share/nginx/html/`.
+The `Deployment` consists of a container image that is listening on port `80`.
+The Service exposes the `Deployment` pods on `nodePort: 30061` on all the nodes
+in the Kubernetes cluster.
 
-The pod, therefore, exposes the fetched zone information at
-`/` or `/index.html` HTTP paths.
+The example also includes an `Ingress` manifest (`ingress/nginx.yaml`) that is
+used by `kubemci` to provision the multi-cluster Ingress.
 
-The Service exposes the Deployment pods on port `30061` on all the
-nodes in its Kubernetes cluster.
+## Step 0: Before you begin
 
-The example also includes an Ingress manifest (`ingress/nginx.yaml`)
-that will be used by `kubemci`.
+1. **GCP Project**
 
-# Tutorial
+    Ensure that you have a GCP Project. If you don't have one already,
+    you can create a project by following the instructions at
+    [Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
 
-## 0. Prerequisite
+    After your project is created, run the following commands in your
+    terminal:
 
-### 0.1 GCP Project
+    ```shell
+    PROJECT=<your-project-name>
+    ```
 
-Ensure you that you have a GCP Project. If you don't have one already,
-you can create a project by following the instructions at
-[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+2. **Set up `gcloud`**:
 
-After your project is created, run the following command in your
-terminal:
+    You can install `gcloud` by following the instructions at
+    [Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+    initialize it by running the following commands:
 
-```shell
-PROJECT=<your-project-name>
-```
+    ```shell
+    gcloud init # Set $PROJECT as the default project
+    gcloud auth login
+    gcloud auth application-default login
+    ```
 
-### 0.2 `gcloud` installed and initialized
+3. Set up **`kubectl`**:
 
-You can install `gcloud` by following the instructions at
-[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
-initialize it by running the following commands:
+    You can install `kubectl` by running:
 
-```shell
-gcloud init # Set $PROJECT as the default project
-gcloud auth login
-gcloud auth application-default login
-```
+    ```shell
+    gcloud components install kubectl
+    ```
 
-### 0.3 kubectl
+## 1. Create Kubernetes Clusters
 
-You can install `kubectl` by running:
+`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. (You can check
+your cluster version using the `kubectl version` command.)
 
-```shell
-gcloud components install kubectl
-```
+You will need at least two Kubernetes clusters in two different compute [zones]
```

Comment:
It has to be zones in 2 different regions.
We provide cross region load balancing: https://cloud.google.com/compute/docs/load-balancing/http/cross-region-example